package.json

```diff
@@ -46,7 +47,7 @@
     "url": "git://github.com/mozilla/shield-studies-addon-utils.git"
   },
   "scripts": {
-    "build": "cd webExtensionApis/study && webpack",
+    "build": "cd webExtensionApis/study && webpack && yaml2json schema.yaml -p > schema.json ; node ../../bin/schemaToInterface.js ./schema.json > fakeApi.js ",
```

Comment:
I don't see a `schemaToInterface.js` file, what is the last of the three commands intended to do?

Reply:
this was a github foulup.